docs/source/cudnn_persistent_rnn.rst

```diff
@@ -0,0 +1,9 @@
+.. note::
+
+    If the following conditions are satisfied:
+    1) cudnn is enabled, 
+    2) input data is on the GPU 
+    3) input data has dtype ``torch.float16`` 
+    4) V100 GPU is used,
+    5) input data is not in ``PackedSequence`` format
+    persistent algorithm can be selected to improve performance.  
```

Comment:
This is selected automatically right? If so, why do we even display such a complicated note to the users?!

Reply:
Ok, where do we have performance guidelines? I'm happy to move it there.